sdcm/utils/alternator.py

```diff
@@ -11,6 +12,13 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class WriteIsolation(Enum):
+    ALWAYS_USE_LWT = "always_use_lwt"
+    FORBID_RMW = "forbid_rmw"
```

Comment:
please go to the place in the code using those values, and point them to use this enum

Reply:
fixed